src/main/java/com/example/bookshop/controller/BookingController.java

```diff
@@ -0,0 +1,114 @@
+package com.example.bookshop.controller;
+
+import com.example.bookshop.entity.Booking;
+import com.example.bookshop.entity.Product;
+import com.example.bookshop.entity.Status;
+import com.example.bookshop.pojos.BookingObj;
+import com.example.bookshop.pojos.ProductObj;
+import com.example.bookshop.pojos.StatusObj;
+import com.example.bookshop.pojos.UserObj;
+import com.example.bookshop.service.IBookingService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/booking")
+public class BookingController {
+    @Autowired
+    private IBookingService bookingService;
+
+
+    /**
+     * Create new booking in DB
+     * @param status of booking
+     * @param description - String
+     * @return a JSON with created booking
+     */
+    @PostMapping("/create")
+    @ResponseStatus(HttpStatus.CREATED)
+    public Booking createBooking(@RequestParam StatusObj status,
+                                 @RequestParam String description,
+                                 @RequestParam ProductObj product,
+                                 @RequestParam UserObj user){
+        return bookingService.createBooking(new BookingObj(status, description, product, user));
+    }
+
+
+    /**
+     * Create one new booking in DB
+     * @param bookingObj Object
+     * @return a JSON with created booking
+     */
+    @PostMapping("/createBooking")
+    @ResponseStatus(HttpStatus.CREATED)
+    public Booking createBooking(@RequestBody BookingObj bookingObj) {
```

Comment:
A bit unclear why we have 2 methods to create bookings

Reply:
Just as example for me demonstrates what I can use @RequestParam or @RequestBody in parameters